Makefile

```diff
@@ -783,14 +784,17 @@ vm-operator-checkout:
 	@if [ -z "${VM_OPERATOR_VERSION}" ]; then echo "VM_OPERATOR_VERSION is not set"; exit 1; fi
 	@if [ -d "$(VM_OPERATOR_TMP_DIR)" ]; then \
 		echo "$(VM_OPERATOR_TMP_DIR) exists, skipping clone"; \
-		cd "$(VM_OPERATOR_TMP_DIR)"; \
-		if [ "$$(git describe --match "v[0-9]*")" != "$(VM_OPERATOR_VERSION)" ]; then \
-			echo "ERROR: checked out version $$(git describe --match "v[0-9]*") does not match expected version $(VM_OPERATOR_VERSION)"; \
-			exit 1; \
-		fi \
 	else \
-		git clone --depth 1 --branch "$(VM_OPERATOR_VERSION)" "https://github.com/vmware-tanzu/vm-operator.git" "$(VM_OPERATOR_TMP_DIR)"; \
+		git clone "https://github.com/vmware-tanzu/vm-operator.git" "$(VM_OPERATOR_TMP_DIR)"; \
+		cd "$(VM_OPERATOR_TMP_DIR)"; \
+        git checkout "$(VM_OPERATOR_COMMIT)"; \
+        git apply "../test/infrastructure/vm-operator/Fix_defaulting_for_Named_networks.patch"; \
```

Comment:
TMP trick while https://github.com/vmware-tanzu/vm-operator/pull/462 merge

Reply:
So we'll be able to get rid of this patch eventually, right?

Just wondering because our patch is different than the vm-operator PR

Reply:
Sounds good. Is the patch here and the PR equivalent? (because the code is different)

Reply:
If we can cherry pick from main, the local patch will be dropped
If we can't cherry pick from main, I will re align patches as soon as the PR merge

Reply:
patch removed